torch/utils/cpp_extension.py

```diff
@@ -2383,6 +2439,12 @@ def _prepare_ldflags(extra_ldflags, with_cuda, verbose, is_standalone):
             else:
                 extra_ldflags.append(f'-L{_join_rocm_home("lib")}')
                 extra_ldflags.append('-lamdhip64')
+    if with_sycl:
+        if IS_WINDOWS:
+            extra_ldflags.append('c10_xpu.lib')
+            extra_ldflags.append('torch_xpu.lib')
```

Comment:
Actually, we also have an additional `torch-xpu-ops.lib` that may need to be linked.